src/Components/ImagesTable/ImagesTable.scss

```diff
@@ -0,0 +1,19 @@
+.pf-c-table {
+    --pf-c-table--m-compact__expandable-row-content--PaddingTop: 1rem;
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
Can you explain this a bit more? How does this maintain the default padding?

Reply:
Specifically we only want to set the padding-left for medium and larger viewports. If we set the padding-left for a small viewport the UUID column will have too much padding and it would look like this: 
![image](https://user-images.githubusercontent.com/95542540/164038404-07fb4467-e513-42dc-b84b-ceaf173144f7.png)

Reply:
Also related, and something I had wanted to ask about, the left padding is *pretty* close to the mockups but not exactly. I have the same question as above with the top padding, how close should we aim to be? @katierik what do you think?
This PR:
![image](https://user-images.githubusercontent.com/95542540/164039019-c5bd5491-deaf-49cd-814b-97f531de05a8.png)
The mockup:
![image](https://user-images.githubusercontent.com/95542540/164039911-64321e6f-aa83-4c72-b0bc-c82d6a313187.png)